lib/ProductOpener/Store.pm

```diff
@@ -332,4 +607,55 @@ sub sto_iter ($initial_path, $pattern = qr/\.sto$/i) {
 	};
 }
 
+=head2 store_config ($path, $ref, $delete_old = 1)
+
+Serializes configuration information, removing it from legacy storage if it is present.
+JSON keys are sorted and indentation is used so files can be used in source control
+No locking is performed
+
+=cut
+
+sub store_config ($path, $ref, $delete_old = 1) {
+	my $sto_path = $path . '.sto';
+	my $file_path = $path . '.json';
+
+	#11901: Config files aren't shared so ignore get_serialize_to_json_level() flag. Just remove this comment when migration is complete
```

Comment:
I might be missing something, but as `store_config` is needed to build taxonomies, should it not write both file types for `get_serialize_to_json_level() > 0`?